src/layouts/EditHomepage/utils.ts

```diff
@@ -0,0 +1,7 @@
+import _ from "lodash"
+
+export const getErrorValues = (
+  obj: Record<string, string>
+): Record<string, string> => {
+  return _.mapValues(obj, () => "")
```

Comment:
for err val we just return empty string for all keys?

Reply:
apparently so - see [here](https://github.com/isomerpages/isomercms-frontend/blob/develop/src/layouts/EditHomepage.jsx#L53)